### ISSUE TITLE
foreach rule doesn't work for arrays in object property

### ISSUE THREAD

**Issue by mobidev111** [Gillespie59/eslint-plugin-angular#476]
eslint setting:
```
/*eslint angular/foreach: 2*/
```

code example:
```
a.b.forEach(function(c, i) {
}
```

expected: 
should show error: You should use the angular.forEach method

actual:
no error

**Comment by Gillespie59** [comment]
Thanks for this report. If you want to solve it, do not hesitate to open a PR. I can be easy to solve this issue.
